lib/spaceship/client.rb

```diff
@@ -66,8 +69,14 @@ def self.hostname
     end
 
     def initialize
+      options = {
+       request: {
+          timeout:       300,
```

Comment:
yup. Up from a default of 60. I noticed that many requests took about 200 seconds to be answered, and with the 60 seconds we ended up retrying too early. It often didn't help...

I would love to understand why this is happening, if we are either doing something wrong, if backend is spinning down/up instances, if the way we upload screenshot causes lots of stress on their architecture (maybe it waits for the screens to be fully processed, I don't know).